### PR TITLE
clean BaseRepository

### PR DESCRIPTION
**Pull request by AlessandroMinoccheri**

### CODE REVIEW

File: src/Mado/QueryBundle/Repositories/BaseRepository.php [hunk 1]
```diff
@@ -10,8 +10,6 @@
 use Symfony\Component\HttpFoundation\Request;
 use Mado\QueryBundle\Queries\QueryBuilderOptions;
 use Mado\QueryBundle\Queries\QueryBuilderFactory;
-use Symfony\Component\DependencyInjection\ContainerAwareInterface;
-use Symfony\Component\DependencyInjection\ContainerAwareTrait;
```
Comment (sensorario) [inline src/Mado/QueryBundle/Repositories/BaseRepository.php]: 👍

File: src/Mado/QueryBundle/Repositories/BaseRepository.php [hunk 2]
```diff
@@ -384,8 +372,8 @@ public function join($queryBuilder, $key, $val)
             $embeddedFields = explode('.', $key);
             $numFields = count($embeddedFields);
 
-            $prevEntityAlias = $this->entityAlias; // Stocksellouts
```
Comment (sensorario) [inline src/Mado/QueryBundle/Repositories/BaseRepository.php]: 👍

File: src/Mado/QueryBundle/Repositories/BaseRepository.php [hunk 3]
```diff
@@ -466,24 +454,24 @@ protected function relationship($queryBuilder)
      */
     public function onDuplicateUpdate($insertFields, $updateFields)
```
Comment (sensorario) [inline src/Mado/QueryBundle/Repositories/BaseRepository.php]: Have you ever used this method?

**Comment by sensorario** [comment]
I think you can merge this PR.

File: src/Mado/QueryBundle/Repositories/BaseRepository.php (hunk 3 above)
Comment (AlessandroMinoccheri) [reply src/Mado/QueryBundle/Repositories/BaseRepository.php]: No but I think that for now It's better to don't remove It

**Pull request merged**